sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -1026,8 +1025,7 @@ void Scheduler::GraphBuilder::cleanupFinishedCommands(
     // Collect stream objects for a visited command.
     if (Cmd->getType() == Command::CommandType::RUN_CG) {
       auto ExecCmd = static_cast<ExecCGCommand *>(Cmd);
-      std::vector<std::shared_ptr<stream_impl>> Streams =
-          std::move(ExecCmd->getStreams());
+      std::vector<std::shared_ptr<stream_impl>> Streams = ExecCmd->getStreams();
```

Comment:
These are to correct this:
windows_pi.cpp : error: implicit conversion between pointer-to-function and pointer-to-object is a Microsoft extension [-Werror,-Wmicrosoft-cast]
  return GetProcAddress((HMODULE)Library, FunctionName.c_str());